lib/color.js

```diff
@@ -563,9 +632,136 @@ class Color {
     }
 }
 
+/*
+ * Sync all color attributes
+ * NOTE: behavior can be disable by setting the 'color_sync' device/group option
+ * @param {Object} newState state with only the changed attributes set
+ * @param {Object} oldState state from the cache with all the old attributes set
+ * @param {Object} options meta.options for the device or group
+ * @return {Object} state with color, color_temp, and color_mode set and syncronized from newState's attributes
+ *                  (other attributes are not included make sure to merge yourself)
+ */
+function syncColorState(newState, oldState, options) {
+    const colorTargets = [];
+    const colorSync = (options && options.hasOwnProperty('color_sync')) ? (options.color_sync == true) : true;
```

Comment:
@sjorge checking this now with zigbee2mqtt tests, but many fail due to color_sync being disabled by default. I don't have a strong opinion here, but what was the reason to disable it by default (since it was previously enabled by default)

Reply:
Does that line mean if we have color_sync and it’s true, we set it to true, otherwise false, when not set, it’s also true?

Reply:
For default true it should be `const colorSync = (options && options.hasOwnProperty('color_sync')) ? options.color_sync : true;`